core/pom.xml

```diff
@@ -192,6 +195,32 @@
                                 </aggregation>
                             </aggregations>
                         </resourcesSet>
+                        <resourcesSet>
+                            <includes>
+                                <include>keynote/lib/reveal.js</include>
+                                <include>keynote/widget.js</include>
+                            </includes>
+                            <aggregations>
+                                <aggregation>
+                                    <inputDir>${resources.dir.compressed}</inputDir>
+                                    <outputFile>${resources.dir.compressed}/keynote/keynote.js</outputFile>
+                                    <withoutCompress>${withoutCompress}</withoutCompress>
+                                </aggregation>
+                            </aggregations>
+                        </resourcesSet>
+                        <resourcesSet>
+                            <includes>
+                                <include>keynote/lib/**/*.css</include>
+                                <include>keynote/theme/*.css</include>
```

Comment:
Won't this accidentally combine all the themes into one giant CSS file?  If this is correct that is OK but makre sure you test switching themes after building with `mvn clean install -Pcompress`

Reply:
I will not, so I remove the line 214.